packages/clay-icon/src/ClayIcon.soy

```diff
@@ -6,11 +6,23 @@
 {template .render}
 	{@param spritemap: string}
 	{@param symbol: string}
+	{@param? ariaLabel: string}
+	{@param? decorative: bool}
 	{@param? elementClasses: string}
 	{@param? focusable: bool}
 	{@param? id: string}
 	{@param? title: string}
 
+	{let $containerAttributes kind="attributes"}
```

Comment:
To follow the same patter the attributes variable of the first element is called just `attributes`, so this one should be `attributes` and the other `svgAttributes`.